app/controllers/contributions_controller.rb

```diff
@@ -20,6 +20,7 @@ def index
 
   def show
     contribution = Listing.find(params[:id])
+    @peer_to_peer = peer_to_peer_mode?
```

Comment:
I tried to pass this in as a local variable (like contribution is passed on line 28) to follow the pattern here more closely but couldn't. It was not accessible from the view. So I settled with this "@" variable. Is there a preference for one or the other?